src/components/ec-smart-table/ec-smart-table.spec.js

```diff
@@ -9,101 +9,200 @@ describe('EcSmartTable', () => {
-          columns, data, sorts, multiSort: false,
+        const wrapper = await mountEcSmartTableWithResolvedData(data, {
+          columns, sorts, multiSort: false,
         });
         expect(wrapper.findByDataTest('ec-table-head').element).toMatchSnapshot('Initial state ([ASC, null, null])');
-        sortColumnByIndex(wrapper, 2);
+        await sortColumnByIndex(wrapper, 2);
         expect(wrapper.findByDataTest('ec-table-head').element).toMatchSnapshot('After sorting different column ([null, null, ASC])');
-        sortColumnByIndex(wrapper, 0);
+        await sortColumnByIndex(wrapper, 0);
         expect(wrapper.findByDataTest('ec-table-head').element).toMatchSnapshot('After sorting different column #2 ([ASC, null, null])');
       });
     });
 
     describe('multi sort', () => {
-      it('should sort column', () => {
+      it('should sort column', async () => {
         const sorts = [
           { column: 'test1', direction: SortDirection.ASC },
         ];
-        const wrapper = mountEcSmartTable({
-          columns, data, sorts, multiSort: true,
+        const wrapper = await mountEcSmartTableWithResolvedData(data, {
+          columns, sorts, multiSort: true,
         });
         expect(wrapper.findByDataTest('ec-table-head__cell--0').element).toMatchSnapshot('ASC');
-        sortColumnByIndex(wrapper, 0);
+        await sortColumnByIndex(wrapper, 0);
         expect(wrapper.findByDataTest('ec-table-head__cell--0').element).toMatchSnapshot('ASC -> DESC');
-        sortColumnByIndex(wrapper, 0);
+        await sortColumnByIndex(wrapper, 0);
         expect(wrapper.findByDataTest('ec-table-head__cell--0').element).toMatchSnapshot('DESC -> not sorted');
-        sortColumnByIndex(wrapper, 0);
+        await sortColumnByIndex(wrapper, 0);
         expect(wrapper.findByDataTest('ec-table-head__cell--0').element).toMatchSnapshot('not sorted -> ASC');
       });
 
-      it('should allow sorting multiple columns', () => {
+      it('should allow sorting multiple columns', async () => {
         const sorts = [
           { column: 'test1', direction: SortDirection.ASC },
         ];
-        const wrapper = mountEcSmartTable({
-          columns, data, sorts, multiSort: true,
+        const wrapper = await mountEcSmartTableWithResolvedData(data, {
+          columns, sorts, multiSort: true,
         });
         expect(wrapper.findByDataTest('ec-table-head').element).toMatchSnapshot('Initial state ([ASC, null, null])');
-        sortColumnByIndex(wrapper, 2);
+        await sortColumnByIndex(wrapper, 2);
         expect(wrapper.findByDataTest('ec-table-head').element).toMatchSnapshot('After sorting different column ([ASC, null, ASC])');
-        sortColumnByIndex(wrapper, 0);
+        await sortColumnByIndex(wrapper, 0);
         expect(wrapper.findByDataTest('ec-table-head').element).toMatchSnapshot('After sorting different column #2 ([DESC, null, ASC])');
       });
     });
   });
 });
+
+function flushPromises() {
```

Comment:
I saw in Vue-test-utils a similar thing but using an npm package flush-promises, but that great do that here directly :)